uint64 calibration_id

string[]                  parent_id
string[]                  child_id
geometry_msgs/Transform[] transform

---

uint8 STATUS_OK=0
uint8 STATUS_ERROR=1

uint8  status
string message
